api/src/test/java/org/openmrs/logic/DurationTest.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.logic;

```

Comment:
You are missing the header here, which is why the test is failing. Please add it